src/shaderlib/fp64/math-fp64.glsl

```diff
@@ -104,9 +104,11 @@ vec2 nint_fp64(vec2 a) {
 
 #if defined(NVIDIA_EQUATION_WORKAROUND) || defined(INTEL_EQUATION_WORKAROUND)
 vec2 twoSum(float a, float b) {
-  float s = (a + b) * ONE;
-  float v = (s - a);
-  float err = (a - (s - v) * ONE) * ONE + (b - v);
+  float s = (a + b);
+  float v = (s * ONE - a) * ONE;
+  float err = (a - (s - v) * ONE) * ONE * ONE * ONE + (b - v);
```

Comment:
This is not exactly intuitive. How is someone else going to be able to maintain this? I think needs a comment, (maybe a url to an explanation if you have one, or at least an explanation of what the issues are).

If this is just trial and error, maybe document what has been tried and failed?